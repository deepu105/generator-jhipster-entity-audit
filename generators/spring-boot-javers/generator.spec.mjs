import { before, describe, expect, it } from 'esmocha';

import { helpers, lookups } from '#test-utils';

const SUB_GENERATOR = 'spring-boot-javers';
const SUB_GENERATOR_NAMESPACE = `jhipster-entity-audit:${SUB_GENERATOR}`;

describe('SubGenerator spring-boot-javers of entity-audit JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create(SUB_GENERATOR_NAMESPACE)
        .withOptions({
          reproducible: true,
          defaults: true,
          localConfig: {
            baseName: 'jhipster',
          },
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
