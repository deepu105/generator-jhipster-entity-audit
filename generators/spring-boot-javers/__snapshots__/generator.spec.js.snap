// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SubGenerator spring-boot-javers of entity-audit JHipster blueprint > run > should succeed 1`] = `
{
  ".yo-rc.json": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/GeneratedByJHipster.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/audit/JaversAuthorProvider.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/config/audit/package-info.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/package-info.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/JaversEntityAuditResource.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/dto/EntityAuditAction.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/dto/EntityAuditEvent.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/dto/package-info.java": {
    "stateCleared": "modified",
  },
  "src/main/java/com/mycompany/myapp/web/rest/package-info.java": {
    "stateCleared": "modified",
  },
}
`;
